templates/referent/_layout.html.twig

```diff
@@ -68,5 +74,29 @@
                 {% block referent_content %}{% endblock %}
             </div>
         </section>
+
+        {% if not app.user.isChartAccepted %}
+            {% include 'chart_dialog.html.twig' with {
+                'title': 'referent.popup.title'|trans,
+                'subtitle': 'referent.popup.subtitle'|trans,
+                'fileUrl': 'referent.popup.file_url'|trans
```

Comment:
I think it will be better to stock these URLs in the config file, `twig->globals` for example. WDYT?

Reply:
In a logic of multi-language website and because the file is wrote in French, I thought it was better to add it there. 
But it could remain in this file and the key can be added in the twig globals @ottaviano ?